src/gtk/textctrl.cpp

```diff
@@ -1226,12 +1263,14 @@ static gboolean afterLayout(void* data)
 void wxTextCtrl::WriteText( const wxString &text )
 {
     wxCHECK_RET( m_text != nullptr, wxT("invalid text ctrl") );
-
+    auto old = m_maxLengthAllowed;
```

Comment:
Use a more descriptive variable name

Reply:
@MaartenBent ,
You have something in mind?

Reply:
Something like `oldMaxLengthAllowed`. So it describes what the variables is...